app/stores/BlockchainStore.js

```diff
@@ -28,6 +28,9 @@ class BlockchainStore {
 
     onGetBlock(block) {
         if (!this.blocks.get(block.id)) {
+            if (!/Z$/.test(block.timestamp)) {
+                block.timestamp + 'Z';
```

Comment:
This needs a `+=` or `block.timestamp = block.timestamp + 'Z'` to actually change the timestamp.